src/main/webapp/i18n/zh-cn/global.json

```diff
@@ -102,7 +102,8 @@
       "id": "ID"
     },
     "select": {
-      "selectOne": "选择"
+      "selectOne": "选择",
+      "selectUser": "选择一个用户"
     },
     "ribbon": {
       "dev": "Development"
```

Comment:
this one not translated

Reply:
As I didn't encounter this contenKey in any event module. Hence I will not change it now as I'm not sure what is/will it use for and which interpretation will suit it purpose.

Reply:
Perhaps the other translations beside than event module I will create another PR. As I don't want this PR to consist too much changed files.